### ISSUE TITLE
Cube Draft - Need a way to turn a deck into a cube

### ISSUE THREAD

**Issue by spjspj** [magefree/mage#2045]
May add an extra item to the lblDraftCube ('Draft Cube') drop down box and another file selection box under that.
![bob](https://cloud.githubusercontent.com/assets/16917728/16540989/0e0c8b1c-40bb-11e6-9b84-fdae8c7e6f27.gif)

Then a new deck cube type object: 
```
package mage.tournament.cubes;
import mage.game.draft.DraftCube;
public class CubeFromDeck extends DraftCube {

public CubeFromDeck(String dckFile) {
        super("Chosen Deck Cube"); 
        foreach (cardName in dckFile) {
            cubeCards.add(new CardIdentity("Act of Treason", ""));
        }
```
or similar

**Comment by LevelX2** [comment]
Related existing tickets:
https://github.com/magefree/mage/issues/1924

**Comment by spjspj** [comment]
Hopefully implemented with 26d38f0773b97a9fe1d5bd089f6be497ce4c2e1f
